generic/tests/cfg/linux_stress.cfg

```diff
@@ -1,6 +1,17 @@
 - linux_stress:
     only Linux
     type = linux_stress
-    stress_duration = 60
-    # Please specify below params for stress test, or use default value as shown
-    # download_url = http://people.seas.harvard.edu/~apw/stress/stress-1.0.4.tar.gz
+    stress_type = "stressapptest"
+    make_cmds_stressapptest = "./configure && make && make install"
+    uninstall_cmds_stressapptest = "./configure && make && make uninstall"
+    download_url_stressapptest = "https://github.com/stressapptest/stressapptest.git"
+    download_type_stressapptest = "git"
+    download_type_stress = "git"
+    stress_cmds_stressapptest = "stressapptest -s 100 -M 256 -m 8 -W"
+    stress_type_vm2 = "stress"
```

Comment:
this should be,
```
stress_type_vm2 = "stress-ng"
```

Reply:
Since params were parsed properly I had left it as-is. Will change to avoid confusion.